### PR TITLE
#184597347: Update dependencies

### PR DESCRIPTION
**Pull request by HzANut**

### CODE REVIEW

**Comment by praves77** [comment]
@XinyuLuan you have been running build off this PR for quite sometime successfully. Correct?

**Review by praves77 — APPROVED** [review]

**Comment by praves77** [comment]
Looks fine to me. will wait for @CodeYouMust to review before merging.

**Comment by CodeYouMust** [comment]
Thx @XinyuLuan .. looks like you have already verified that all versions are supported on .net framework `v4.5.1` and have tested changes? if so.. didnt have any other feedback.

**Pull request merged**
